multi-area-model.ipynb

```diff
@@ -0,0 +1,260 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b1331599",
+   "metadata": {},
+   "source": [
+    "# Down-scaled multi-area model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96517739",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e07b0d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install nested_dict dicthash"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c6b6a3c",
+   "metadata": {},
+   "source": [
+    "Create config file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "72c170e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('config.py', 'w') as fp:\n",
+    "    fp.write(\n",
+    "'''import os\n",
+    "base_path = os.path.abspath(\".\")\n",
+    "data_path = os.path.abspath(\"simulations\")\n",
+    "jobscript_template = \"python {base_path}/run_simulation.py {label}\"\n",
+    "submit_cmd = \"bash -c\"\n",
+    "''')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2784f76b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from multiarea_model import MultiAreaModel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cedd26b",
+   "metadata": {},
+   "source": [
+    "Neurons and indegrees are both scaled down to 1%.\n",
+    "Can usually be simulated on a local machine.\n",
+    "\n",
+    "**Warning: This will not yield reasonable dynamical results from the\n",
+    "network and is only meant to demonstrate the simulation workflow.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e940bb6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scale_down_to = 0.005"
```

Comment:
Here you set the number of neurons and indegrees to .5 %, but above you state you would set it to 1%. Changing either one should be fine.

Reply:
Thanks for pointing me to this mistake. It is fixed now.